### PR TITLE
Remove examples in hipblaslt

### PR DESCRIPTION
**Pull request by KKyang**
Remove deprecated examples

### CODE REVIEW

File: clients/benchmarks/CMakeLists.txt [hunk 1]
```diff
@@ -92,3 +92,53 @@ set_target_properties( hipblaslt-bench PROPERTIES
...
+foreach( exe ${ext_bench_list_all} )
+  target_link_libraries( ${exe} PRIVATE hip::device )
+endforeach( )
+
+foreach( exe ${ext_bench_list_all} )
+  rocm_install(TARGETS ${exe} COMPONENT samples)
+endforeach( )
+
+# Add tests for special cases
+add_test(NAME hipblaslt-bench-groupedgemm-fixed-mk
```
Comment (jichangjichang) [inline clients/benchmarks/CMakeLists.txt:143]: what is this ?

File: clients/benchmarks/CMakeLists.txt [hunk 2]
```diff
@@ -92,3 +92,53 @@ set_target_properties( hipblaslt-bench PROPERTIES
 add_dependencies( hipblaslt-bench hipblaslt-common )
 
 rocm_install(TARGETS hipblaslt-bench COMPONENT benchmarks)
+
+# Other bench files
+add_executable( hipblaslt-bench-groupedgemm-fixed-mk client_groupedgemm_fixed_mk.cpp)
```
Comment (jichangjichang) [inline clients/benchmarks/CMakeLists.txt:97]: suggest following the file name

**Review by jichangjichang — APPROVED** [review]
LGTM

File: clients/benchmarks/CMakeLists.txt (hunk 2 above)
Comment (KKyang) [reply clients/benchmarks/CMakeLists.txt:97]: I followed the hipblaslt-bench with client.cpp and hipblaslt-bench

**Review by Serge45 — APPROVED** [review]
LGTM.

**Pull request merged**